src/components/Organisms/Main/Main.css

```diff
@@ -25,17 +25,27 @@
   max-width: 1200px;
 }
 
+.mainContainer:after {
+  content: '';
+  display: block;
+  clear: both;
+}
+
 @media bp-maxWidth {
   .mainContainer {
     margin: 1rem auto 2rem;
   }
 }
 
-.mainList,
 .mainList2,
 .latestContent,
 .callouts {
-  margin-bottom: 2rem;
+  margin-top: 2rem;
```

Comment:
Any reason for the switch to margin-top versus margin-bottom? There's no issue here (although I do try and keep all the margins running in the same - making sure to keep consistent with one or the other will help with debugging). Just wanted to make sure I'm not missing some secret that's better when using top.

Reply:
Just used margin-top to work with the `* + *` selects. I'll refactor to use margin-bottom and lose that slow selector, as noted below.